test_utility.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import utility
+
+# testing with already existing uuid
```

Comment:
still need a better way to unit test

Reply:
U mean having multiple parameters in one function? Regarding that, I will implement and make a new pull request.

Reply:
Please write a ` #TODO`